job_definitions/index_services.yml

```diff
@@ -9,8 +9,8 @@
     parameters:
       - string:
           name: INDEX
-          default: g-cloud
-          description: "Index name to use (eg 'g-cloud-2017-05-22')"
+          default: {{ default_index_name }}
```

Comment:
The search-api already takes the index name as a parameter, so the change we're planning to make is actually on the buyer-frontend. But essentially, yes.

The logic for doing this is explained a bit more in comments on <https://trello.com/c/dhIpjg15/618-data-modelling-documentation>

In summary, we need to be able to derive a framework name given a search index name, and the simplest way to do that is to guarantee that the index is named for (one of) the framework(s) it relates to.